host-components/external/enarx-keep@.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Enarx Keep-Loader
+Documentation=https://github.com/enarx/
+
+[Service]
+Type=simple
+Environment="SCRIPT_ARGS=%I"
+ExecStart=/home/mike/programming/enarx/host-components/target/debug/keep-loader $SCRIPT_ARGS
+UMask=0066
+
+StandardOutput=file:/home/mike/keeploader.log
```

Comment:
hmm, I guess you want `append:`, otherwise every instance might overwrite the others?
Or maybe you want:
```
StandardOutput=file:/home/mike/keeploader-%i.log
```

Reply:
Don't think this will get fixed in time for the end of the Sprint, so new issue here for all systemd-related questions: https://github.com/enarx/enarx/issues/849

Reply:
Done - this produces rather ugly looking files, but at least they're separate ones.